apiTester/src/commonMain/kotlin/com/revenuecat/purchases/kmp/apitester/PurchasesAreCompletedByAPI.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases.kmp.apitester
+
+import com.revenuecat.purchases.kmp.PurchasesAreCompletedBy
+import com.revenuecat.purchases.kmp.StoreKitVersion
+
+@Suppress("unused")
+private class PurchasesAreCompletedByAPI {
+    fun check(mode: PurchasesAreCompletedBy) {
+        when (mode) {
+            is PurchasesAreCompletedBy.RevenueCat -> { }
+            is PurchasesAreCompletedBy.MyApp -> {
+                mode.storeKitVersion
```

Comment:
Could be good to assign to a variable to verify the type:
```suggestion
                val storeKitVersion: StoreKitVersion  = mode.storeKitVersion
```

Reply:
Love this! I applied your suggestion in commit https://github.com/RevenueCat/purchases-kmp/pull/139/commits/ae097bbafd6abb97346c7b83be07062b1e58e351